devel/contents.tex

```diff
@@ -48,10 +48,16 @@ \section{Examination}%
 The report will be graded according to the following grading criteria:
 \begin{description}
   \item[Grade E] You fulfil all the \acp{ILO} at the minimum level.
```

Comment:
What is minimum level? All the ILO? Must they also apply MSBs methodological support in the project?
Nothing else? What is the minimum expectancy of this project assignment? They should have reviewed research articles and applied them to some extent? What is the minimum extent for a pass? Purely theoretical application? A practical application but with gaps or lack of their own empirical data? Quantitative? 1-2 articles per group member? Is it enough that they applied parts of the articles?

Reply:
No MSB, I should put the ILOs explicitly there.

The expectancy is that they should state a relevant problem and convince us that they are equipped to solve it adequately. An A means they are perfect for solving such problems. An E means they know how to do it, their solutions will not be the best, but they are not total crap either (basically, they should have taken security and usability into account in their solution).

Reply:
\item[Grade E] You fulfil the set \acp{ILO} for the assignment,
You should have identified a relevant problem, and given a solution to it.
It must be a viable solution, however gaps and misstakes are allowed, if they don't render your solution unusable.

Reply:
I realized when I talked through the project with the instruction that the ILOs are in the intro of the instruction :-)

Sounds good. Still lots of subjectivity in there :-P